### PR TITLE
Task 010/create basic UI

### PR DESCRIPTION
**Pull request by GRPilot**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,7 +33,11 @@ find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets LinguistTools REQUIRED)
 find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets LinguistTools REQUIRED)
 
 load_directories_list(${PROJECT_SOURCE_DIR}/src directories)
-load_sources_list(${PROJECT_SOURCE_DIR}/src sources)
+load_sources_list(${PROJECT_SOURCE_DIR}
+    sources
+    translate_files
+    resources_files
```
Comment (seigtm) [inline CMakeLists.txt:39]: resource_files would be better

File: src/mainwindow/mainwindow.cpp [hunk 2]
```diff
@@ -1,15 +1,45 @@
...
+    } else {
+        showMaximized();
+    }
+}
+
+void MainWindow::onMinimizeButtonClicked() {
+    showMinimized();
+}
+
+void MainWindow::onMenuToggled(bool checked) {
```
Comment (seigtm) [inline src/mainwindow/mainwindow.cpp:43]: wanna const bool "just because" 😙

File: src/mainwindow/mainwindow.ui [hunk 3]
```diff
@@ -1,22 +1,489 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
+ <author>Konstantin Baranov</author>
```
Comment (seigtm) [inline src/mainwindow/mainwindow.ui:3]: Ruslan Golovinsky too?

File: src/mainwindow/mainwindow.ui [hunk 4]
```diff
@@ -1,22 +1,489 @@
...
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
-    <height>600</height>
+    <width>1162</width>
```
Comment (seigtm) [inline src/mainwindow/mainwindow.ui:10]: Maybe lower default width to support really small screens?

**Review by seigtm — COMMENTED** [review]
Some minor remarks from me, but it's still kinda sus ngl 🧐

File: src/mainwindow/mainwindow.cpp (hunk 2 above)
Comment (GRPilot) [reply src/mainwindow/mainwindow.cpp:43]: It's slot's definition by qt so idk, but I'll try later

File: src/mainwindow/mainwindow.ui (hunk 3 above)
Comment (GRPilot) [reply src/mainwindow/mainwindow.ui:3]: You're a creator and designer. I'd just refactored some details

File: src/mainwindow/mainwindow.ui (hunk 4 above)
Comment (GRPilot) [reply src/mainwindow/mainwindow.ui:10]: Yeah. I'll run our the-best-assembled application on my 90s pocket computer. It's so important to me! 👨🏿‍💻

**Pull request merged**